src/pages/graphql/payment-methods/payment-services-minicart.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Payment Services product page workflow
+description: Learn how to use the GraphQL API mutations for the Payment Services solution.
+keywords:
+  - GraphQL
+  - Payments
+---
+
+# Payment services product page workflow
+
+![Payment Services sequence diagram](../../_images/graphql/payment-services-minicart.svg)
+
+These steps describe the additional flow of requests and responses required to place an order after [adding a product to your cart](../tutorials/checkout/add-product-to-cart.md) with the [Payment Services](https://experienceleague.adobe.com/docs/commerce-merchant-services/payment-services/guide-overview.html){target=_blank} solution enabled.
+
+1. Fetch the payment configuration by running the `getPaymentConfig` mutation in order to render payment methods.
+
+1. Adobe Commerce returns available payment configuration.
+
+1. Creates the order with the `createPaymentOrder` mutation.
+
+1. Request forwarded to PayPal.
+
+1. PayPal returns an `id` value.
```

Comment:
I am not sure if the meaning is lost on the receiving side. 

The client receives the " PayPal `id` " but paypal is returning  an "`id`".
Can we say that PayPal is returning a `PayPal id`.